src/lib/components/Modal/ModalHead.jsx

```diff
@@ -0,0 +1,83 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {
+  RUIContext,
+  withGlobalProps,
+} from '../../provider';
+import styles from './ModalHead.scss';
+
+export const ModalHead = ({
```

Comment:
In https://github.com/react-ui-org/react-ui/blob/master/src/lib/components/Card/README.mdx we refer to the `Card` _header_. I think we should keep this the same across all components. I think I like `Header` better, but I don't really care.

Reply:
I don't care as well. Let it discuss it on Monday with @adamkudrna

Reply:
@adamkudrna ?

Reply:
I agree, let's go with `ModalHeader`.